src/main/resources/alps-core.yaml

```diff
@@ -692,6 +720,17 @@ definitions:
     items:
       $ref: '#/definitions/Device'
 
+  DeviceUpdate:
+    type: object
+    description: Describes update of device, it allows to change name of device and device token (only in case of mobile devices)
+    properties:
+      name:
+        type: string
+        description: New device name (optional)
+      deviceToken:
+        type: string
+        description: Token used for pushing matches. The token needs to start with `apns://` or `fcm://` dependent on the device or channel the match should be pushed with
```

Comment:
`The token needs to start with` =>  `The token needs to be prefixed with`
to indicate that the token you are getting from fcm/apns should be prefixed
maybe provide some example?